### ISSUE TITLE
More helpful list of timezones for .env configuration

### ISSUE THREAD

**Issue by cathalgarvey** [huginn/huginn#2599]
I'm trying to use Docker to deploy a personal instance, so the suggestion to just "use rake" isn't super helpful. If I download a local copy of the repo anyway and run `rake` or `rake time:zones:local` as instructed, I get `...LoadError: cannot load such file -- bundler/setup...` along with a traceback. I'm sure this would be more meaningful if I were a Ruby person. :)

Basically I just want to know how to add my timezone, because it's a formatted rich-text field instead of a UTC offset or standard TZ code. And I can't find a list, and nothing useful comes up if I search the git repo. So, it'd be more ergonomic to either default to some external standard that can be referenced easily, or to have a documented list of acceptable timezone names.

Thanks!

**Comment by cathalgarvey** [comment]
I think I may have sourced the mapping/list here, for others sharing my confusion: https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html

**Comment by dsander** [comment]
You are right, that is indeed not very helpful, would you be interested in sending a PR that updates the documentation?
